chart/values.schema.json

```diff
@@ -5363,6 +5368,11 @@
                     ],
                     "default": null
                 },
+                "terminationGracePeriodSeconds": {
+                    "description": "Grace period for webserver to finish after SIGTERM is sent from Kubernetes.",
+                    "type": "integer",
+                    "default": 30
```

Comment:
in all current deployment this value is 60.
why you choose 30? (apply for others)

Reply:
I meant that the terminationGracePeriodSeconds in other components equal to 60. 
[tigerrer](https://github.com/apache/airflow/blob/main/chart/values.yaml#L1579)
[dag processor](https://github.com/apache/airflow/blob/main/chart/values.yaml#L1766)
It's better to keep `terminationGracePeriodSeconds` consistent across all components unless there's a stronger reason to do otherwise.

Reply:
Gotcha, let's keep it like you did.